tools/eslint-plugin/dist/rules/require-version-in-deprecation.js

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+//Custom rule that enforces version and description for @deprecated
+
+"use strict";
+
+module.exports = {
+  meta: {
+    messages: {
+      requireVersionAndSentence: "@deprecated in Major.minor format followed by deprecation reason and/or alternative API usage required"
+    }
+  },
+  create(context) {
+    return {
+      Program(node) {
+        let match;
+        for (const comment of node.comments) {
+          if (match = /@deprecated(?<in> in \d+(\.(\d|x))+.{5,})?/.exec(comment.value)) {
```

Comment:
i question if we should require the description, would be nice to have but not the biggest dealbreaker for me.
pretty much every deprecation comment paul updated in https://github.com/iTwin/itwinjs-core/pull/4935 will need to get updated which is kinda annoying

Reply:
It is what Paul requested: https://github.com/iTwin/itwinjs-core/issues/4968#issuecomment-1404180693 and I think it makes sense.